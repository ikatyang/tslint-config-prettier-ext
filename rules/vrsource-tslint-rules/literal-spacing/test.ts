import {x} from 'x';

const y = {a:1};

const z = [ 0 ];
