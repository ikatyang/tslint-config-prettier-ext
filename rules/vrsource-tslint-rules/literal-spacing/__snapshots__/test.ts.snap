// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[1, 9]: Missing whitespace
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[1, 10]: Missing whitespace
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[3, 12]: Missing whitespace
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[3, 15]: Missing whitespace
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[5, 12]: Found extra space
ERROR: (literal-spacing) /rules/vrsource-tslint-rules/literal-spacing/test.ts[5, 14]: Found extra space

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
import {x} from 'x';

const y = {a:1};

const z = [ 0 ];

======
import { x } from \\"x\\";

const y = { a: 1 };

const z = [0];

>>>>>> after
"
`;
